runtime/common/src/lib.rs

```diff
@@ -117,6 +115,34 @@ pub mod types {
 	#[derive(codec::Encode, codec::Decode, Default, Clone, PartialEq, Eq, TypeInfo)]
 	#[cfg_attr(feature = "std", derive(Debug))]
 	pub struct EthAddress(pub Bytes32);
+
+	/// Rate with 27 precision fixed point decimal
+	pub type Rate = crate::fixed_point::Rate;
+
+	/// Amount with 18 precision fixed point decimal
+	pub type Amount = crate::fixed_point::Amount;
+
+	/// PoolId type we use.
+	pub type PoolId = u64;
+
+	/// A representation of ClassId for Uniques
+	pub type ClassId = u64;
+
+	/// A representation of InstanceId for Uniques.
+	#[derive(
+		codec::Encode,
+		codec::Decode,
+		Default,
+		Copy,
+		Clone,
+		PartialEq,
+		Eq,
+		codec::CompactAs,
+		Debug,
+		TypeInfo,
+	)]
+	#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+	pub struct InstanceId(pub u128);
```

Comment:
Do we wanna have a common `TrancheId` and `EpochId` to, if the rest is here?

Reply:
yes for sure. Maybe a seperate ticket?